pkg/scaffold/update.go

```diff
@@ -20,7 +20,7 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/model"
```

Comment:
I think we can fully remove the `update` command right @mengqiy?

Reply:
Or at least remove it's logic and check `updateCmd.HasSubCommands()` before adding it, like kubebuilder does with `alpha`.

Reply:
It seems this file is no longer needed.

Reply:
I agree but IHMO we should remove the update command in a follow-up PR. 
Then, if we need for any reason to do a revert will be easier. 
The scope here is NOT to remove commands or deprecated args. The goal here is just clean up and remove the v1 code source.